x/ibc/07-tendermint/misbehaviour.go

```diff
@@ -71,12 +71,14 @@ func checkMisbehaviour(
 
 	// assert that trustedVals is NextValidators of last trusted header
 	// to do this, we check that trustedVals.Hash() == consState.NextValidatorsHash
-	tvalHash := evidence.Header1.TrustedValidators.Hash()
-	if !bytes.Equal(consensusState.NextValidatorsHash, tvalHash) {
+	trustedValsetHash1 := evidence.Header1.TrustedValidators.Hash()
+	trustedValsetHash2 := evidence.Header2.TrustedValidators.Hash()
+
+	if !(bytes.Equal(consensusState.NextValidatorsHash, trustedValsetHash1) || bytes.Equal(consensusState.NextValidatorsHash, trustedValsetHash2)) {
```

Comment:
```suggestion
	if !(bytes.Equal(consensusState.NextValidatorsHash, trustedValsetHash1) && bytes.Equal(consensusState.NextValidatorsHash, trustedValsetHash2)) {
```

hmm if one is true it returns false, I think this should be an && ?

Reply:
ie if at least one doesn't have equal bytes the if should be triggered

Reply:
we should add a test in a followup for this